homeworks/hw12/hw12.txt

```diff
@@ -0,0 +1,21 @@
+Напишіть CSS або XPath селектори для доданого файла:
+
+1 - Select the first <h2> element:
+
+//CSS - h2:first-of-type
+//XPath - //h2[1]
+
+2 - Select all list items under the <ul> element with the id "fruits":
```

Comment:
2 - Select all list items under the <ul> element with the id "fruits":

//CSS - ul[id = fruits] *   OR ul#fruits li
//XPath - //ul[@id="fruits"]/li